Microsoft.Toolkit.Parsers/Markdown/MarkdownDocument.cs

```diff
@@ -13,14 +13,17 @@
 using System;
 using System.Collections.Generic;
 using System.Text;
-using Microsoft.Toolkit.Uwp.UI.Controls.Markdown.Helpers;
+using Microsoft.Toolkit.Parsers.Markdown.Blocks;
+using Microsoft.Toolkit.Parsers.Markdown.Enums;
+using Microsoft.Toolkit.Parsers.Markdown.Helpers;
 
-namespace Microsoft.Toolkit.Uwp.UI.Controls.Markdown.Parse
+namespace Microsoft.Toolkit.Parsers.Markdown
 {
     /// <summary>
-    /// A class used to represent abstract markdown.
+    /// Represents a Markdown Document. <para/>
+    /// Initialize an instance and call <see cref="Parse(string)"/> to parse the Raw Markdown Text.
     /// </summary>
-    internal class MarkdownDocument : MarkdownBlock
+    public class MarkdownDocument : MarkdownBlock
```

Comment:
Could be nice if the constructor can also take a string and just pass that to the Parse method.

Is there a reason this doesn't have the IParser interface too?

Reply:
IParser is used for schema based parsers, like some XML, and RSS. Markdown doesn't really fit this.

The Constructor with no string input, and a void Parse method, mimics the style of XDocument and XmlDocument parsing in .NET itself, so I think it should stay that way.